src/lib.js

```diff
@@ -677,6 +676,40 @@ var program,
       }
     },
 
+    /**
+     * Identifies whether the input data string is a graphql query or not
+     *
+     * @param {string} dataString - Input data string to check if it is a graphql query
+     * @returns {Object} - { result: true, graphql: {Object} } if dataString is a graphql query else { result: false }
+    */
+    identifyGraphqlRequest: function (dataString) {
+      try {
+        const rawDataObj = _.attempt(JSON.parse, dataString.replace(/\r\n/g, ''));
+        if (rawDataObj && !_.isError(rawDataObj)) {
+          if (!_.has(rawDataObj, 'variables')) {
+            rawDataObj.variables = {};
+          }
+          if (_.keys(rawDataObj).length === 2 && _.has(rawDataObj, 'query') && _.has(rawDataObj, 'variables')) {
+            if (typeof rawDataObj.query === 'string' && typeof rawDataObj.variables === 'object') {
+              return {
+                result: true,
+                graphql: {
+                  query: rawDataObj.query,
+                  // eslint-disable-next-line max-len
```

Comment:
Rather than adding custom rule here, we should fix the actual error. i.e. You can store `JSON.stringify(rawDataObj.variables, null, 2)` as variable as it's used 2 times anyways and use that variable for shorter line.